### PR TITLE
feat: add format-json

### PR DESCRIPTION
**Pull request by leozhantw**
Hi cloudingcity,

This tool works very well for me, but I think it would be better if it had the feature **format-json**.

So I made this PR!

### CODE REVIEW

**Comment by cloudingcity** [comment]
Hi @leozhantw,
Thank you for your contribute.
This is a good idea.
But I think just use `github.com/tidwall/pretty` package handle this would be better.
That what I did in [jwt-decode](https://github.com/cloudingcity/gool/blob/master/cmd/jwt.go#L22) command

File: cmd/prettyjson.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import (
+	"fmt"
+
+	"github.com/cloudingcity/gool/pkg/prettyjson"
+	"github.com/spf13/cobra"
+)
+
+var prettyJSONCmd = &cobra.Command{
+	Use:                   "pretty-json [content]",
```
Comment (cloudingcity) [inline cmd/prettyjson.go:11]:
```suggestion
	Use:                   "format-json [text]",
```
And put in format.go

Maybe we will have format-xml format-sql cmd.

File: cmd/prettyjson.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	Short:                 "Input the content of JSON",
```
Comment (cloudingcity) [inline cmd/prettyjson.go:12]:
```suggestion
	Short:                 "Cleans and format JSON",
```

**Review by cloudingcity — CHANGES_REQUESTED** [review]
Please update the scripts block of [README.md](https://github.com/cloudingcity/gool/blob/master/README.md#scripts).

File: cmd/prettyjson.go (hunk 1 above)
Comment (leozhantw) [reply cmd/prettyjson.go]: Great.

I have been done for this suggestion

File: cmd/prettyjson.go (hunk 2 above)
Comment (leozhantw) [reply cmd/prettyjson.go]: I have been done for this suggestion~!

**Comment by leozhantw** [comment]
> Hi @leozhantw,
> 
> Thank you for your contribute.
> 
> This is a good idea.
> 
> But I think just use `github.com/tidwall/pretty` package handle this would be better.
> 
> You can see what I did in [jwt-decode](https://github.com/cloudingcity/gool/blob/master/cmd/jwt.go#L22) command



> Hi @leozhantw,
> 
> Thank you for your contribute.
> 
> This is a good idea.
> 
> But I think just use `github.com/tidwall/pretty` package handle this would be better.
> 
> You can see what I did in [jwt-decode](https://github.com/cloudingcity/gool/blob/master/cmd/jwt.go#L22) command

**Comment by leozhantw** [comment]
> Hi @leozhantw,
> 
> Thank you for your contribute.
> 
> This is a good idea.
> 
> But I think just use `github.com/tidwall/pretty` package handle this would be better.
> 
> You can see what I did in [jwt-decode](https://github.com/cloudingcity/gool/blob/master/cmd/jwt.go#L22) command

Oh my gineus!

I have been done for this suggestion!

File: cmd/format.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	"github.com/tidwall/pretty"
+)
+
+var formatJSONCmd = &cobra.Command{
+	Use:                   "format-json [text]",
+	Short:                 "Cleans and format JSON",
+	Args:                  cobra.ExactArgs(1),
+	DisableFlagsInUseLine: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(string(pretty.Pretty([]byte(args[0]))))
```
Comment (cloudingcity) [inline cmd/format.go:16]: How do you think print with color?
```suggestion
		fmt.Print(string(pretty.Color(pretty.Pretty([]byte(args[0])), nil)))
```

Comment (leozhantw) [reply cmd/format.go]: Great!

I have been done for this suggestion~!

**Pull request merged**